### ISSUE TITLE
Unable to import component: No module named 'parse'

### ISSUE THREAD

**Issue by ubarni** [mampfes/ha_bayernluefter#5]
Since the last release i get the following error when starting the integration:

```
2024-01-26 04:40:55.251 ERROR (MainThread) [homeassistant.setup] Setup failed for custom integration 'bayernluefter': Unable to import component: No module named 'parse'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 251, in _async_setup_component
    component = integration.get_component()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 822, in get_component
    ComponentProtocol, importlib.import_module(self.pkg_path)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/config/custom_components/bayernluefter/__init__.py", line 21, in <module>
    from .pyernluefter import Bayernluefter
  File "/config/custom_components/bayernluefter/pyernluefter/__init__.py", line 6, in <module>
    import parse
ModuleNotFoundError: No module named 'parse'
```

**Comment by mampfes** [comment]
fixed in version 1.2.3

**Comment by ubarni** [comment]
Just installed 1.2.3. Bug fixed 👍🏼
